code/datums/components/art.dm

```diff
@@ -49,3 +49,37 @@
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artgreat", /datum/mood_event/artgreat)
 	else
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artbad", /datum/mood_event/artbad)
+
+// NT/syndicate themed art, currently used in posters
+
+/datum/component/art/nt
+
+/datum/component/art/nt/on_obj_examine(datum/source, mob/M)
+	apply_moodlet(M, 0)
```

Comment:
Why are you overriding this proc

Reply:
so it doesn't ask for impressiveness, see what the rev bust does